src/lang/en.js

```diff
@@ -12,12 +12,20 @@ export default {
     },
     general: {
       search: 'Search',
-      requiredInline: '(required)', // Appended to required form field labels
+      // Appended to required form field labels
+      requiredInline: '(required)',
+      yes: 'yes',
```

Comment:
Is this lowercase on purpose? If so, may be useful for translators if you provide a comment saying that and maybe why

Reply:
🤔 
It appears after a filter. For example on orders "Refunded: yes". Do you think I should:
- Put a comment saying it's used after other text like that example
- Put the comment in, and also move it out of "general" into "filters"
- Make it capitalised anyway (so it's "Refunded: Yes"
- Capitalise it, but transform it to lower case

Reply:
#1